.github/workflows/maven-deploy.yaml

```diff
@@ -0,0 +1,19 @@
+name: Publish package to the Internal Maven Repository
+on:
+  workflow_call:
+    inputs:
+      distribution:
```

Comment:
This parameter is not used, right? If not, then let's remove it

Reply:
Yup, removed.